regulations/generator/layers/formatting.py

```diff
@@ -44,37 +40,36 @@ def render_fence(self, fence):
             lines = [l.replace('Note:', '').replace('Notes:', '')
                      for l in lines]
             lines = [l for l in lines if l.strip()]
-            tpl = self.note_tpl
-        elif _type == 'extract':
+            tpl = self.tpls['note']
+        elif _type == 'extract':    # @todo can be removed soon
```

Comment:
@tadhg-ohiggins recently shifted `EXTRACT`s from a fenced text, which didn't allow for tables, headers, etc. into sub trees. The reference to `extract` type, then, can go away once everything settles down